components/dashboard/src/components/PrebuildLogs.tsx

```diff
@@ -129,11 +129,13 @@ export default function PrebuildLogs(props: PrebuildLogsProps) {
                     <WorkspaceLogs classes="h-full w-full" logsEmitter={logsEmitter} errorMessage={error?.message} />
                 </Suspense>
             </div>
-            <div className="h-20 px-6 bg-gray-50 dark:bg-gray-800 border-t border-gray-200 dark:border-gray-600 flex space-x-2">
-                {prebuild && <PrebuildStatus prebuild={prebuild} />}
-                <div className="flex-grow" />
-                {props.children}
-            </div>
+            {prebuild && (
```

Comment:
:bulb: Looking at this code made me :sparkles: the reason why we do not have a prebuild (and thus: skip button) here when triggering a manual one: Because at render time, there is no prebuild yet, because we're about to create it :see_no_evil: .

So to solve the prebuild issues, we should either a) poll for the prebuild if we do not find any, or b) make sure we have hooks in place to ge updated whenenver this happens.

Anyway, sth for a follow-up. :+1:

Reply:
Will create a follow-up issue for this.